linting/.editorconfig

```diff
@@ -135,9 +135,9 @@ dotnet_style_prefer_simplified_interpolation = true:warning
 [*.{cs,csx,cake}]
 # Implicit and explicit types
 # https://docs.microsoft.com/visualstudio/ide/editorconfig-language-conventions#implicit-and-explicit-types
-csharp_style_var_for_built_in_types = true:warning
+csharp_style_var_for_built_in_types = never
 csharp_style_var_when_type_is_apparent = true:warning
-csharp_style_var_elsewhere = true:warning
+csharp_style_var_elsewhere = false:warning
```

Comment:
Why are these rules changed?

Reply:
I’ll dig out the original issue on Roslyn but essentially there’s a bug which means it allows `var` when the type is not apparent on the right hand side.